INSTALL.md

```diff
@@ -0,0 +1,269 @@
+#Installation Instructions
+
+OSI Certified Open Source Software
+
+-------------------------------------
+
+### Table of Contents
+
+**[Overview of Directories](#overview-of-directories)**
+
+**[Unpacking](#unpacking)**
+
+**[Setup](#setup)**
+
+**[Setting up Access Control](#setting-up-access-control)**
+
+**[Upgrading](#upgrading)**
+
+**[FAQ](#faq)**
+
+
+##  Overview of Directories
+
+NOTE: Most recent documentation can be found on the online documentation at [OpenEMR](http://www.open-emr.org).
+
+accounting: Contains information and scripts to support SQL-Ledger
+contrib: Contains many user-contributed encounter forms and utilities
+custom: Contains scripts and other text files commonly customized
+Documentation: Contains useful documentation
+interface: Contains User Interface scripts and configuration
+library: Contains scripts commonly included in other scripts
+sql: Contains initial database images
+gacl: Contains embedded php-GACL (access controls)
+
+##  Unpacking
+
+NOTE: Most recent documentation can be found on the online documention at [OpenEMR](http://www.open-emr.org).
+
+The LibreEHR release archive should be named as follows:
+
+`libreehr-<version>.tar.gz  -or-  libreehr-<version>.zip`
+
+To extract the archive, use either of the following commands from the command line:
+
+`bash# tar -pxvzf libreehr-<version>-release.tar.gz`
+`bash# unzip libreehr-<version>-release.tar.gz`
+
+Be sure to use the `-p` flag when using tar, as certain permissions must be preserved.
+
+LibreEHR will be extracted into a directory named `libreehr`.
```

Comment:
The folder name should be kept shorter, I guess? I'll name it `libre-health-ehr` in the next commit, can change it later if need be.